i18next-parser.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  lexers: {
+    js: ['JsxLexer'],
+  },
+  defaultNamespace: 'translation',
+  locales: ['en', 'jp'],
+  output: './src/i18n/translations/$LOCALE/translation.json',
+  input: ['src/**/!(Trans)*.js'],
```

Comment:
```suggestion
  input: ['src/**/*.js'],
```

You can tweak this slightly since we don't have a custom `Trans` component in this repo. I was encountering errors from this tool when trying to extract translations, so I ignored it as part of the input.

Reply:
oh gotcha. derp!